src/messages.js

```diff
@@ -30,6 +30,12 @@ export default {
   'option.locationTermStatuses.accepted': 'active',
   'option.locationTermStatuses.rejected': 'inactive',
 
+  // Movement
+
+  'field.movements_common.locationDate.name': 'Inventory/move date',
+  'field.movements_common.currentLocation.fullName': 'Storage location',
+  'inputTable.movement.currentLocation': 'Storage location',
+
```

Comment:
Missed changes:

- Change record name and collectionName to "Inventory/Movement"
- Change 'Object Location Information' panel label to "Inventory/Movement Information"
- Change 'Reason for move' field to 'Reason'
- Change 'Movement method' field to 'Method'